hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -8,8 +8,8 @@ filesets:
   files_rtl:
     depend:
       - lowrisc:ip:tlul
+      - lowrisc:ip:lc_ctrl_pkg
     files:
-      - rtl/lc_ctrl_pkg.sv
```

Comment:
FYI, I had to remove this since it otherwise collides with the already existing `lc_ctrl_pkg`. For instance, this broke our VCS regression. I'd rather suggest we add the "real" `lc_ctrl_pkg` to bronze instead.

Reply:
Yes, adding the pkg the bronze was also my initial suggestion.
So once this PR is merged to master, and the lc_ctrl_pkg is added to v1-bronze we can push this PR to bronze as well.